arch/arm/src/tlsr82/Kconfig

```diff
@@ -250,6 +250,53 @@ menuconfig TLSR82_USB
 menuconfig TLSR82_PWM
 	bool "PWM Configuration"
 	default n
+	select PWM
+
+if TLSR82_PWM
+
+# PWM0 configuration
+
+	config TLSR82_PWM0
```

Comment:
I think other paces do not have such alignment. Maybe we can remove TAB suffix in this case as well?

Reply:
Ack and Done.